Manifest.toml

```
@@ -1,8 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.9.2"
+julia_version = "1.9.4"
```

Can you try that with julia 1.10 ?
```suggestion
julia_version = "1.10"
```